gordon/record_checker.py

```diff
@@ -27,41 +27,40 @@
 class RecordChecker(object):
 
     def __init__(self, dns_ip):
-        """Setup RecordChecker object
+        """Setup RecordChecker object.
 
         Args:
-            dns_ip: DNS ip server to query
+            dns_ip: DNS server IP to query.
         """
         self._dns_ip = dns_ip
         self._resolver = ProxyResolver()
         try:
             self._resolver.set_proxies([self._dns_ip])
         except async_dns.address.InvalidHost as e:
-            msg = f'RecordChecker got invalid DNS server ip: {e}'
+            msg = f'RecordChecker got invalid DNS server IP: {e}'
             raise exceptions.InvalidDNSHost(msg)
 
     def _extract_record_data(self, record):
-        """Extract record data from a record dict
+        """Extract record data from a record dict.
 
         Args:
-            record (dict): DNS record as a dict
+            record (dict): DNS record as a dict.
 
         Returns:
-            The record data: name, rrdata, type and ttl
+            The record data: name, rrdata, type and ttl.
         """
         return record.values()
 
     async def check_record(self, record, timeout=60):
         """Measures the time for a DNS record to become available.
 
-        Query a provided DNS server multiple time until
+        Query a provided DNS server multiple times until
             the reply matches the information in the
             record or until timeout is reached.
```

Comment:
while you're at it, do you want to fix this indentation? i believe it's also incorrect.